pkg/certificates/certificates_test.go

```diff
@@ -736,7 +736,7 @@ func TestNewClient(t *testing.T) {
 
 	tlsOptions, err := tlsopts.NewOptions(ident, tlsopts.Config{})
 	require.NoError(t, err)
-	clientTransport := transport.NewClient(tlsOptions)
+	clientTransport := transport.NewClientWithTimeout(tlsOptions, 20*time.Millisecond)
```

Comment:
Is there a reason we are not using default values here?

Reply:
not really, I just thought it shouldn't take longer than that amount of time but I guess we're not explicitly testing for that here. I'll take it out.